src/EA.Weee.Xml/MemberRegistration/v3schema.xsd

```diff
@@ -59,7 +60,7 @@
 	<xs:simpleType name="XSDVersionType">
 		<xs:restriction base="xs:decimal">
 			<xs:minInclusive value="3.06"/>
```

Comment:
I'm not sure why this was changed to allow 3.06 (it was lowered, having been 3.07, back in 2015) -- so I'm not sure whether this should be raised to 3.07 or 3.08, or left as is.